### DISCUSSION TITLE
Fake Beacon Broadcast

### DISCUSSION THREAD

**Discussion by ghost** [nzymedefense/nzyme#591] [Ideas]
Hello,

I think a good feature to add (or create a new project) is broadcast fake stations, APs, and SSIDs. This can make it more difficult for an attacker to attack the real APs, stations, and SSIDs in use. 

The idea is that the wireless card on a Raspberry with Nzyme device to utilize the unused built-in wireless card. Instead of just going un-used, why not use it for further defense. The Raspberry Pi 4 has both 2.4 and 5 Ghz built-in and would be great to utilize this if not being used for connecting to the real network or monitoring. 

Making it more difficult for the attacker to determine what may or may not be real devices in the wireless environment can be the difference of an attacker conducting a quick scan and choosing a single target vs the attacker being presented with numerous APs, stations, and devices and perhaps having to take more time to conduct a successful attack or at best simply move on and not even try. **If an attacker can't easily see their target, if they don't know what the real target looks like, it's more difficult for them to perform an attack**. 

Nzyme wireless monitoring is great but I'm hoping for more. I hope the concept of broadcasting fake stations, APs, or SSIDs can also be implemented. This can make it more difficult for determined attackers that want to cause harm to be successful. 

Thanks,

Jason

**Comment by lennartkoopmann** [comment]
Hi! Thank you. Have you looked at the [Deception and Traps](https://www.nzyme.org/docs/deception-traps/intro) already built into nzyme?

**Reply by ghost** [reply]
wow, this is exactly what I'm looking for. I will look into this on how to implement it.

**Reply by lennartkoopmann** [reply]
Awesome! Let me know if you get stuck somewhere and if you have any ideas for improvements or further traps.
